src/server/api/routes/user.router.js

```diff
@@ -23,6 +23,18 @@ router.get("/", authorizeUser(ROLES.SUPER_USER), (req, res, next) => {
     .catch(next);
 });
 
+// ENDPOINT: /api/users/:role :GET
```

Comment:
Fowad, i was talking with @zkwsk about the endpoint and that should be /api/user-roles/:user_role_id/users

Reply:
I'm sorry if I said something wrong. I see that it was also not described in the Trello at all. But Rahim is right about the path.

Reply:
I have created the new router and now modified the right path.
please have a look.